geostructures/geohash.py

```diff
@@ -8,8 +8,8 @@
 ]
 
 import abc
-from collections import defaultdict, Counter
-from typing import Dict, List, Optional, Sequence, Set, Tuple, TypedDict
+from collections import Counter, defaultdict

```

Comment:
Is `Counter` still getting used?

Reply:
It was being used in hash_coordinates (I don't think the checks would let me have an unused import), but no reason not to expand agg_fn syntax to hash_coordinates, so now I've done that and dropped Counter.